python/app/input_widget.py

```diff
@@ -439,8 +436,7 @@ def _current_line_color(self):
 
     def _format_exc(self):
         """Get the latest stack trace and format it for display."""
-        tb = sys.exc_info()[2]
-        return traceback.format_exc(tb)
```

Comment:
Bugfix: I don't think the traceback should ever have been passed through to this method as the `limit` kwarg.

Reply:
I'm confused. What are you saying here exactly?

Reply:
[`traceback.format_exec`](https://docs.python.org/2/library/traceback.html#traceback.format_exc) only supports a limit kwarg. We were passing a traceback object through as if it was a limit, and that seemed to work in Python 2. The traceback object could be used with a less or more than operator. In Python 3 it can't, but I think this was a bug anway in the original code anyway.

Reply:
oh, get it!